<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frameCorRGB">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>241</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLCDNumber" name="lcdNumberR">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumberG"/>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumberB"/>
         </item>
        </layout>
       </widget>
       <widget class="Plotter" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>7</y>
          <width>239</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderR">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderG">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderB">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelMinimoX">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderX">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelMinimoY">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderY">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelMinimoZ">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderZ">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>215</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="radioButtonPutVoxel">
       <property name="text">
        <string>Colocar Voxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCutVoxel">
       <property name="text">
        <string>Retirar Voxel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonIniciar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSalvar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="PlotterMain" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>941</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPaint_3D">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionAjuda"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuPaint_3D"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo"/>
   <addaction name="actionSalvar"/>
   <addaction name="actionAjuda"/>
   <addaction name="actionSair"/>
  </widget>
  <action name="actionSalvar_arquivo">
   <property name="text">
    <string>Salvar arquivo</string>
   </property>
  </action>
  <action name="actionNovo_arquivo">
   <property name="text">
    <string>Novo arquivo</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="icon">
    <iconset resource="ArquivosDeRecursos.qrc">
     <normaloff>:/icones/icones/novo.png</normaloff>:/icones/icones/novo.png</iconset>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="ArquivosDeRecursos.qrc">
     <normaloff>:/icones/icones/salvar.png</normaloff>:/icones/icones/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="icon">
    <iconset resource="ArquivosDeRecursos.qrc">
     <normaloff>:/icones/icones/ajuda.jpg</normaloff>:/icones/icones/ajuda.jpg</iconset>
   </property>
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="ArquivosDeRecursos.qrc">
     <normaloff>:/icones/icones/sair.png</normaloff>:/icones/icones/sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>setColorRed(int)</slot>
    <slot>setColorGreen(int)</slot>
    <slot>setColorBlue(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>PlotterMain</class>
   <extends>QWidget</extends>
   <header>plottermain.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ArquivosDeRecursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberR</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>819</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberG</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>864</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberB</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>923</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>948</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setColorRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setColorGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>893</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget_2</receiver>
   <slot>setColorBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>962</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
